grammar/haxe.bnf

```diff
@@ -383,7 +383,7 @@ blockStatement ::= '{' statementList? '}' {mixin="com.intellij.plugins.haxe.lang
 private statementList ::= (statement ';'?)+ {recoverWhile="statement_recovery"}
 private statement_recovery ::= !('case' | 'default' | '}')
 
-private statement ::= blockStatement | notBlockStatement
+private statement ::= macroClassList? (blockStatement | notBlockStatement)
```

Comment:
The pin has to move from the default position of 1.  So add {pin=2} to the end of the rule or on the next line.  Otherwise a macro, if it exists will match the rule and whether it's a statement or a value is ambiguous (so the first rule wins).  Thus, between the two changes, most values will be parsed as statements and will create a host of misleading parsing errors.

Did the unit tests actually pass with this change?